apps/testing-javascript/src/pages/index.tsx

```diff
@@ -22,6 +22,12 @@ import type {TestimonialProps, FaqProps, InterviewProps} from '@types'
 
 import LandingTemplate from 'templates/landing-template'
 
+const testingJavaScriptProductIds = [
```

Comment:
imo this should be dynamic instead of hard coded in general but it's OK for here probanly